### DISCUSSION TITLE
Weird thing when trying to connect to azure using github for the first time

### DISCUSSION THREAD

**Discussion by msjio** [community/community#78504] [GitHub Education] [answered]
This issue is not just me but my friends too.
hi thanks for this amazing pack , but when I try to redeem azure, I need to connect using my GitHub {I tried using other normal email to connect but didn't work at all} I tried to change the email and deleting the school one it did connect me but it gave me a notice that this other email is not the one eligible for the GitHub student pack and I should apply for it too again even Tho my account is approved.
when I try to connect using my normal email of school"myemail" I get this error :
The email "my email"
![Capture JPG](https://github.com/community/community/assets/149756822/f4f6b0ff-c342-4f89-a751-31a676544e48)
![Capture1 JPG](https://github.com/community/community/assets/149756822/72c0336c-498e-472e-b0f9-d63011cfaad9)
![Capture2 JPG](https://github.com/community/community/assets/149756822/5316d8fe-6ea2-4883-9992-d8d0abfff836)
![Capture3 JPG](https://github.com/community/community/assets/149756822/ffa4d7e1-3de5-48d2-9f77-5c45e3d1f5c9)
 in your GitHub profile belongs to your work or school. You cannot use GitHub to sign in.
then when I try to login/signing using it even Tho I don't have any account associated with it in a normal way it says :
Sign in
This username may be incorrect. Make sure you typed it correctly. Otherwise, contact your admin

**Comment by ft10101** [comment]
try using different email while sign up

**Reply by msjio** [reply]
i did so but when i do that it says that im not eligible for the students azure subscription.

**Reply by ft10101** [reply]
use gmail or some other email, then try again

**Reply by k0ns0l** [reply]
Hello @ft10101, the issue still persists even after signing with a different email provider (google). Do you have any suggestion as a work-around this issue?

**Reply by ft10101** [reply]
No. Try contacting GitHub support and mail them your issue maybe they can help

**Reply by k0ns0l** [reply]
A CS resolution with the Microsoft team is under way!

Thank you!

**Reply by msjio** [reply]
@k0ns0l how did you fix it?

**Reply by k0ns0l** [reply, marked as answer]
@msjio I created a support ticket on this [page](https://azure.microsoft.com/en-us/free/students/).
Though it took quite some time, I'd say a week :) but nevertheless, they sent a verification ID for the students' subscription.

You'd be asked some supporting materials - screenshot of active students' membership.

If all is verified, the CS team will escalate your issue to the Engineers, and hopefully it gets fixed.

**Reply by msjio** [reply]
@k0ns0l thank you for answering. I will do that.
